openslides/agenda/static/js/agenda/site.js

```diff
@@ -268,13 +269,19 @@ angular.module('OpenSlidesApp.agenda.site', [
                     return false;
             }
         };
-        $scope.getUpdateStatePrefix = function (item) {
+        $scope.getDetailStatePrefix = function (item) {
             var prefix = item.content_object.collection.replace('/','.');
             // Hotfix for Issue 2566.
             // The changes could be reverted if Issue 2480 is closed.
```

Comment:
Issue 2480 is closed. A pull request was merged yesterday or the day before. Please check if this hotfix is still necessary.